src/quantize.rs

```diff
@@ -284,30 +289,25 @@ impl QuantizationContext {
     // To that end, we want to bias more toward rounding to zero for
     // that tail of zeroes and ones than we do for the larger coefficients.
     let mut level_mode = 1;
-    for (qc, c) in
-      qcoeffs[1..].iter_mut().zip(coeffs[1..].iter()).take(last_pos)
-    {
-      let coeff = *c << self.log_tx_scale;
+    for &pos in scan[1..].iter().take(last_pos) {
```

Comment:
I just noticed I have some confusing naming conflicts I need to fix in this patch. Should change the earlier pos and last_pos.